### ISSUE TITLE
error: bundling: UnableToResolveError: Unable to resolve module `AccessibilityInfo` from ...

### ISSUE THREAD

**Issue by JanaTrajkovska** [facebook/react-native#14276]

**Comment by SupriyaKalghatgi** [comment]
Why is this issue closed without any solution or reference to other issues?

**Comment by hramos** [comment]
Because the issue is completely empty and is non-actionable.
